commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/ExportSubCommand.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.commandline.subcommands;
+
+import static tech.pegasys.web3signer.slashingprotection.SlashingProtectionFactory.createSlashingProtection;
+
+import tech.pegasys.web3signer.core.InitializationException;
+import tech.pegasys.web3signer.slashingprotection.SlashingProtection;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.HelpCommand;
+import picocli.CommandLine.MissingParameterException;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
+
+@Command(
+    name = "export",
+    description = "Exports the slashing protection database",
+    subcommands = {HelpCommand.class},
+    mixinStandardHelpOptions = true)
+public class ExportSubCommand implements Runnable {
```

Comment:
Do these seperate export/import subcommands need to be registered as subcommands in Pico? I couldn't see where that is happening.

Reply:
They are declaratively specified - in the @Command section of the Eth2SubCommand